README.md

```diff
@@ -15,7 +15,7 @@ Mumble is an Open Source, low-latency and high-quality voice-chat program
 written on top of Qt and Opus.
 
 There are two modules in Mumble; the client (mumble) and the server (murmur).
-The client works on Windows, Linux, FreeBSD, OpenBSD and macOS,
+The client works on Windows, Linux, FreeBSD, OpenBSD, and macOS,
```

Comment:
I don't think commas belong in front of an "and"

Reply:
We should add a comma before 'and' when it connects two independent clauses.

Reply:
Both versions are correct here, although the expression without a comma before the `and` is more common. 
The "commas don't belong in front of `and`" rule is more of a German thing and not even true there all the time.

Reply:
It's called an oxford comma(or some other names) and is indeed a valid way for lists(I personally do prefer it)

Reply:
Interesting. Never knew about that one ^^